api4/user_test.go

```diff
@@ -2382,56 +2382,145 @@ func TestSetDefaultProfileImage(t *testing.T) {
 	}
 }
 
-func TestCBALogin(t *testing.T) {
+func TestLogin(t *testing.T) {
```

Comment:
I take it, this test just didn't exist before? +1!

Reply:
Yes, that's correct! The diff here is sadly misleading :(